src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/SpatialRelationships/SpatialRelationships.xaml.cs

```diff
@@ -0,0 +1,176 @@
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntime.WPF.Samples.SpatialRelationships
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Spatial relationships",
+        "GeometryEngine",
+        "This sample demonstrates how to use the GeometryEngine to evaluate the spatial relationships (for example, polygon a contains line b) of between geometries.",
+        "Tap a graphic to select it. The display will update to show the relationships with the other graphics.")]
+    public partial class SpatialRelationships
+    {
+        private GraphicsOverlay _graphicsOverlay;
+        private Graphic _polygonGraphic;
+        private Graphic _polylineGraphic;
+        private Graphic _pointGraphic;
+
+        public SpatialRelationships()
+        {
+            InitializeComponent();
+
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Configure the basemap
+            MyMapView.Map = new Map(Basemap.CreateTopographic());
+
+            // Create the graphics overlay and set the selection color
+            _graphicsOverlay = new GraphicsOverlay()
+            {
+                SelectionColor = System.Windows.Media.Colors.Yellow
+            };
+
+            // Add the overlay to the MapView
+            MyMapView.GraphicsOverlays.Add(_graphicsOverlay);
+
+            // Create the point collection that defines the polygon
+            PointCollection polygonPoints = new PointCollection(SpatialReferences.WebMercator)
+            {
+                new MapPoint(-5991501.677830, 5599295.131468),
+                new MapPoint(-6928550.398185, 2087936.739807),
+                new MapPoint(-3149463.800709, 1840803.011362),
+                new MapPoint(-1563689.043184, 3714900.452072),
+                new MapPoint(-3180355.516764, 5619889.608838)
+            };
+
+            // Create the polygon
+            Polygon polygonGeometry = new Polygon(polygonPoints);
+
+            // Define and apply the symbology
+            SimpleLineSymbol polygonOutlineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Windows.Media.Colors.Green, 2);
+            SimpleFillSymbol polygonFillSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.ForwardDiagonal, System.Windows.Media.Colors.Green, polygonOutlineSymbol);
+
+            // Create the graphic and add it to the graphics overlay
```

Comment:
Clarify some code comments (done for WPF - applies for the other SDK's as well):
```
// Create the polygon graphic and add it to the graphics overlay
```